app/code/core/Mage/ImportExport/Model/Resource/Helper/Mysql4.php

```diff
@@ -49,11 +56,26 @@ public function getMaxDataSize()
     public function getNextAutoincrement($tableName)
     {
         $adapter = $this->_getReadAdapter();
+        try {
+            $this->setInformationSchemaStatsExpiry();
+        } catch (Exception $e) {
+            //this will throw an exception for < mysql8
+        }
         $entityStatus = $adapter->showTableStatus($tableName);
-
         if (empty($entityStatus['Auto_increment'])) {
             Mage::throwException(Mage::helper('importexport')->__('Cannot get autoincrement value'));
         }
         return $entityStatus['Auto_increment'];
     }
+
+    /**
+     * Set information_schema_stats_expiry to 0 if not already set.
+     */
+    public function setInformationSchemaStatsExpiry(): void
+    {
+        if (!self::$instantInformationSchemaStatsExpiry) {
+            $this->_getReadAdapter()->query('SET information_schema_stats_expiry = 0;');
```

Comment:
```suggestion
            try {
                $this->_getReadAdapter()->query('SET information_schema_stats_expiry = 0;');
            } catch (Exception $e) {}
```

I'd do it this way, so that the try/catch is not even called unless it's used